src/client/components/DiscountTag/DiscountTag.tsx

```diff
@@ -0,0 +1,49 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import { useRedeemedCampaignsQuery } from 'data/graphql'
+import { getDiscountText } from 'pages/OfferNew/Introduction/Sidebar/utils'
+import { OfferData } from 'pages/OfferNew/types'
+import { isNorwegianBundle } from 'pages/OfferNew/utils'
```

Comment:
I think we should try to avoid importing these more specific utilities in a global component. However, I'm sure we do that all over the place so no worries for now. Just a remainder for all of us to avoid it :)